src/main/java/co/rsk/federate/btcreleaseclient/BtcReleaseClient.java

```diff
@@ -289,15 +287,22 @@ public void onBlock(org.ethereum.core.Block block, List<TransactionReceipt> rece
          * </p>
          *
          * @param currentBlockNumber the current block number in the blockchain
-         * @param svpTxHash the Keccak256 hash of the svp spend transaction waiting to be signed
+         * @param svpSpendTx the Keccak256 hash and the Bitcoin transaction of the svp spend transaction waiting to be signed
          * @return {@code true} if the transaction has the required number of confirmations and is ready to be signed;
          *         {@code false} otherwise
          */
         private boolean isSVPSpendTxReadyToSign(long currentBlockNumber, Map.Entry<Keccak256, BtcTransaction> svpSpendTx) {
             try {
+
+                BtcTransaction btcTx = svpSpendTx.getValue();
```

Comment:
```suggestion
                BtcTransaction svpSpendTx = svpSpendTx.getValue();
```

no big deal at all, but its actually the spend tx